packages/typespec-ts/test/modularIntegration/lroStardard.spec.ts

```diff
@@ -0,0 +1,314 @@
+import {
+  StandardClient,
+  User
+} from "./generated/lro/standard/generated/src/index.js";
+import { assert } from "chai";
+import { restorePoller } from "./generated/lro/standard/generated/src/restorePollerHelpers.js";
+import { OperationState } from "@azure/core-lro";
+
+describe("LROStandardClient Classical Client", () => {
+  let client: StandardClient;
+
+  beforeEach(() => {
+    client = new StandardClient({
+      allowInsecureConnection: true
+    });
+  });
+
+  describe("createOrReplace", () => {
+    it("just call without await", async () => {
+      client.createOrReplace("madge", {
+        role: "contributor"
+      } as any);
+    });
+    it("should await poller result directly", async () => {
+      try {
+        const result = await client.createOrReplace("madge", {
+          role: "contributor"
+        } as any);
+        assert.deepEqual(result, { name: "madge", role: "contributor" });
+      } catch (err) {
+        assert.fail(err as string);
+      }
+    });
+
+    it("should count polling counts correctly", async () => {
```

Comment:
I'm not sure I understand what we want to validate here?

Reply:
as we offline discussioned i would delete this case becaues it is covered by core.